daemon/prune.go

```diff
@@ -26,7 +26,8 @@ import (
 var (
 	// errPruneRunning is returned when a prune request is received while
 	// one is in progress
-	errPruneRunning = fmt.Errorf("a prune operation is already running")
+	errPruneRunning   = fmt.Errorf("a prune operation is already running")
+	errPruneCancelled = fmt.Errorf("a prune operation is cancelled")
```

Comment:
I don't think think we need to add this new error here, but in any case the wording is a bit funky.
Maybe `the prune operation was cancelled`